backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
+        first_name: {type: 'varchar(30)'},
```

Comment:
Change limit to 128 chars? There are long ass names in the world =)

Reply:
Agree